docs/source/apps/projinfo.rst

```diff
@@ -218,6 +218,14 @@ The following control parameters can appear in any order:
     For example, `+proj=utm +zone=31 +datum=WGS84 +type=crs` will be identified
     with a likelihood of 70% to EPSG:32631
 
+.. option:: --3d
+
+    .. versionadded:: 7.0
```

Comment:
Woops, this isn't good for a 6.2 backport

Reply:
> Woops, this isn't good for a 6.2 backport

What do you mean ? Should be we remove the --3d option, or fix the versionadded ?

Reply:
6.2.1 is a patch release so we shouldn't introduce new features. But it is a bit tricky because it would be nice to have the auto-3D-promotion in effect since it fixes actual problems. If it is easy to remove the `--3d` option I'm in favour of that, but I release that may make things complicated.

Reply:
projinfo changes reverted

Reply:
Thank you!